StripeCore/StripeCoreTests/Attestation/MockAppAttestService.swift

```diff
@@ -66,10 +67,14 @@ class MockAppAttestService: AppAttestService {
         key.counter += 1
         keys[key.id] = key
         // Generate a fake attestion
-        let attestation = ["keyID": key.id, "counter": key.counter, "clientDataHash": clientDataHash.base64EncodedString()] as [String: Any]
+        let attestation = ["keyID": key.id, "counter": key.counter, "clientDataHash": clientDataHash.base64EncodedString(), "isDevelopmentEnvironment": attestationUsingDevelopmentEnvironment] as [String: Any]
         return try JSONSerialization.data(withJSONObject: attestation)
     }
 
+    @_spi(STP) public func attestationDataIsDevelopmentEnvironment(_ data: Data) -> Bool {
+        let decodedKey = try! JSONSerialization.jsonObject(with: data) as! [String: Any]
+        return decodedKey["isDevelopmentEnvironment"] as! Bool
+    }
```

Comment:
Feels like we can avoid using `as!` and fail gracefully?  E.g. if we can't decode, default to development.

Reply:
This is test code, so I want it to fail loudly